### ISSUE TITLE
[Rails5] Ruby on Rails stub server requires MySQL but does not use it

### ISSUE THREAD

**Issue by ledsun** [OpenAPITools/openapi-generator#254]
##### Description

The Ruby on Rails stub server requires MySQL.
Its Gemfile includes line blew:
```
# Use mysql as the database for Active Record
gem 'mysql2', '>= 0.3.18', '< 0.5'
```

To start the Ruby on Rails stub server, we need to install MySQL in the development environment.
However, the Ruby on Rails stub server does not use the database.
I think this is a meaningless cost.

##### openapi-generator version

3.0.0

##### OpenAPI declaration file content or url

Any.

##### Command line used for generation

```sh
bin/rails5-petstore-server.sh
```

##### Steps to reproduce

```sh
bin/rails5-petstore-server.sh
```

##### Related issues/PRs

No.

##### Suggest a fix/enhancement

Switch the database to SQLite for Ruby on Rails stub server like:
https://github.com/ledsun/openapi-generator/tree/rails_reduce_dependency

We can start the Ruby on Rails stub server in fewer steps.

**Comment by wing328** [comment]
@ledsun thanks for reporting the issue. I agree with you SQLite works better for new users who use the auto-generated Ruby-on-Rails app for the first time.

May I know if you've time to contribute a PR with the enhancement?

**Comment by ledsun** [comment]
I can create a PR today or tomorrow.

**Comment by ackintosh** [comment]
Changed to '3.1.0' as https://github.com/OpenAPITools/openapi-generator/pull/279 is planed to release in v3.1.0.

**Comment by ackintosh** [comment]
@ledsun The enhancement is merged into `3.1.0` branch. ✨ can we close this issue?

**Comment by wing328** [comment]
Sure.
